src/main/resources/api.yaml

```diff
@@ -592,22 +603,36 @@ components:
     SprintSummaryDto:
       type: object
       properties:
+        sprint:
+          $ref: '#/components/schemas/SprintDto'
         membersAvailability:
           type: array
           items:
             $ref: '#/components/schemas/UserAvailabilityDto'
           description: List of members' availability for given sprint
-        totalAvailableTime:
+        prevAverageSprintCoefficient:
+          type: number
+          format: double
+        currentAverageSprintCoefficient:
```

Comment:
Do pól można tutaj chyba dać description, nie? Byłoby super, jakby tutaj się udało dodać parę słów do każdego - ja w miarę ogarniam, które jest które, ale i tak ciągle skaczę do opisu PRa. 

_Radykalna idea_: może dajmy nawet tutaj numerki komórek z arkusza, to frontowi łatwiej będzie się w tym ogarnąć, jak przyjdzie co do czego

Reply:
najchętniej to bym walnął całą treść PR w komentarzu gdzieś, może nawet w osobnym pliku typu readme